infra/services/postgres.bicep

```diff
@@ -0,0 +1,62 @@
+targetScope = 'resourceGroup'
+
+param testApplicationOid string
```

Comment:
Considering that only baseName, location, tenantIdand and testApplicationOid will be supplied, this seems excessive